### DISCUSSION TITLE
Multiple Displays using TFT_eSPI

### DISCUSSION THREAD

**Discussion by cmcooper1971** [Bodmer/TFT_eSPI#1329] [Q&A - General]
Hi,

Is it possible to use TFT_eSPI for multiple displays? When using Adafruit library’s, I invoke the library as follows:

Adafruit_ILI9341 tft0 = Adafruit_ILI9341(TFT_CS, TFT_DC);
Adafruit_ILI9341 tft1 = Adafruit_ILI9341(TFT1_CS, TFT1_DC);
Adafruit_ILI9341 tft2 = Adafruit_ILI9341(TFT2_CS, TFT2_DC);

This doesn’t work with the eSPI library as there isn’t a parameter listed to parse through different chip select pins.

I have watched a video which describes how I need to disable the chip select pin within the user.h file, and then use digitalwrite highs and lows to control each screen, which works, but seems very cumbersome.

Is this the way or am I missing something?

Regards

Christopher

**Comment by Adrianotiger** [comment]
Is there a workaround, maybe to redefine the DC pin before declaring another display?
